2019-11-10 02:14:32,297	INFO	main1	<module>	Line:11	app started
2019-11-10 02:14:32,308	WARNING	_internal	_log	Line:122	 * Debugger is active!
2019-11-10 02:14:32,330	INFO	_internal	_log	Line:122	 * Debugger PIN: 185-856-785
2019-11-10 02:14:32,343	INFO	_internal	_log	Line:122	 * Running on http://192.168.2.210:9100/ (Press CTRL+C to quit)
2019-11-10 02:14:36,528	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 02:14:36] "GET /ProblemSet/fa8624a7-de2a-4507-ba23-85a36f820eda/ww HTTP/1.1" 200 -
2019-11-10 02:14:37,613	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 02:14:37] "GET /ProblemSet/init/fa8624a7-de2a-4507-ba23-85a36f820eda/ww HTTP/1.1" 200 -
2019-11-10 02:14:37,646	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 02:14:37] "GET /ProblemSet/answer/fa8624a7-de2a-4507-ba23-85a36f820eda/3 HTTP/1.1" 200 -
2019-11-10 02:14:39,714	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 02:14:39] "GET /ProblemSet/addscore/fa8624a7-de2a-4507-ba23-85a36f820eda/143/0/1/d424c36bed25d956b783f831a6327d20 HTTP/1.1" 200 -
2019-11-10 02:14:39,735	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 02:14:39] "GET /ProblemSet/answer/fa8624a7-de2a-4507-ba23-85a36f820eda/2 HTTP/1.1" 200 -
2019-11-10 02:14:39,740	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 02:14:39] "GET /ProblemSet/wrongproblem/fa8624a7-de2a-4507-ba23-85a36f820eda/1/3 HTTP/1.1" 200 -
2019-11-10 02:14:43,855	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 02:14:43] "GET /ProblemSet/problemanswer/fa8624a7-de2a-4507-ba23-85a36f820eda HTTP/1.1" 200 -
2019-11-10 02:14:45,637	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 02:14:45] "GET /ProblemSet/finishedtime/fa8624a7-de2a-4507-ba23-85a36f820eda/143 HTTP/1.1" 200 -
2019-11-10 02:14:45,690	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 02:14:45] "GET /ProblemSet/problemanswer/fa8624a7-de2a-4507-ba23-85a36f820eda HTTP/1.1" 200 -
2019-11-10 02:14:45,713	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 02:14:45] "GET /ProblemSet/problemanswer/fa8624a7-de2a-4507-ba23-85a36f820eda/5 HTTP/1.1" 200 -
2019-11-10 02:15:10,775	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 02:15:10] "GET /ProblemSet/addscore/fa8624a7-de2a-4507-ba23-85a36f820eda/143/0/1/d424c36bed25d956b783f831a6327d20 HTTP/1.1" 200 -
2019-11-10 02:15:10,777	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 02:15:10] "GET /ProblemSet/answer/fa8624a7-de2a-4507-ba23-85a36f820eda/1 HTTP/1.1" 200 -
2019-11-10 02:15:10,795	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 02:15:10] "GET /ProblemSet/wrongproblem/fa8624a7-de2a-4507-ba23-85a36f820eda/1/2 HTTP/1.1" 200 -
2019-11-10 02:15:52,091	INFO	_internal	_log	Line:122	 * Detected change in 'C:\\development\\examsystem\\services\\daos\\problemsetDAO.py', reloading
